Updated S4Mk2 manual with proposed changes from PR (#309)
mixxxdj/manual: source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk2.rst

:hwlabel:`FILTER` knob                              Controls QuickEffect superknob. This controls the Filter effect by default, but a different effect can be chosen in the Equalizer section of Mixxs’s Preferences.
:hwlabel:`SNAP`                                     Toggles library fullscreen
:hwlabel:`LOOP RECORD`                              Toggles recording
:hwlabel:`LOOP RECORDER PLAY`                       Hold down and press :hwlabel:`FLUX` button on a deck to enable autoslip mode on that deck.
:hwlabel:`SHIFT` + :hwlabel:`Gain`                  Up/down will move the beatgrid

> :hwlabel:`LOOP RECORDER PLAY` + :hwlabel:`GAIN      Increase / decrease the BPM of the track by 0.5 - press in the knob to round the track the nearest whole BPM. Best done to fix tracks with the wrong BPM with the pitch fader at 0.

Reviewer: ```suggestion
:hwlabel:`LOOP RECORDER PLAY` + :hwlabel:`GAIN`      Increase / decrease the BPM of the track by 0.5 - press in the knob to round the track the nearest whole BPM. Best done to fix tracks with the wrong BPM with the pitch fader at 0.
```

> :hwlabel:`LOOP RECORDER PLAY` + :hwlabel:`GAIN`      Increase / decrease the BPM of the track by 0.5. Press the encoder to round the track the nearest whole BPM. Best done to fix tracks with the wrong BPM with the pitch fader at 0.

Reviewer 2: ```suggestion
:hwlabel:`LOOP RECORDER PLAY` + :hwlabel:`GAIN`     Increase / decrease the BPM of the track by 0.5. Press the encoder to round the track the nearest whole BPM. Best done to fix tracks with the wrong BPM with the pitch fader at 0.
```

> :hwlabel:`BROWSE` encoder                           Up/down will browse in the focused library pane. Pressing acts like a double click on the mouse.

Reviewer 2: Warning: /home/runner/work/manual/manual/source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk2.rst:60: WARNING: Inline interpreted text or phrase reference start-string without end-string.

:hwlabel:`FILTER` knob                              Controls QuickEffect superknob. This controls the Filter effect by default, but a different effect can be chosen in the Equalizer section of Mixxs’s Preferences.
:hwlabel:`SNAP`                                     Toggles library fullscreen
:hwlabel:`LOOP RECORD`                              Toggles recording

> :hwlabel:`LOOP RECORDER PLAY`                       Hold down to use as a modifier.

Reviewer: ```suggestion
:hwlabel:`LOOP RECORDER PLAY`                       Hold down and press :hwlabel:`FLUX` button on a deck to enable autoslip mode on that deck.
```

:hwlabel:`SHIFT` + :hwlabel:`Gain`                  Up/down will move the beatgrid

> :hwlabel:`BROWSE` encoder                           Up/down will browse in the focused library pane. Clicking in acts like a double click on the mouse.

Reviewer: ```suggestion
:hwlabel:`BROWSE` encoder                           Up/down will browse in the focused library pane. Pressing acts like a double click on the mouse.
```
"Click" is an unusual term to use when discussing an encoder

==================================================  =========================================================================
:hwlabel:`LOAD`                                     Load track selected in library to the deck.
:hwlabel:`SHIFT` + :hwlabel:`LOAD`                  Eject track

> Small buttons with preview icons                       Play a sampler from its cue point. If no track is loaded in the sampler, the track selected in the library will be loaded.
> :hwlabel:`SHIFT` + small buttons with preview icons If sampler is playing, stop it. If sampler is not playing, the loaded eject track from the sampler.

Reviewer 3: ```suggestion
Small buttons with preview icons                    Play a sampler from its cue point. If no track is loaded in the sampler, the track selected in the library will be loaded.
:hwlabel:`SHIFT` + small buttons with preview icons If sampler is playing, stop it. If sampler is not playing, the loaded track is ejected from the sampler.
```

:hwlabel:`1-4` numbered buttons                     Set/activate :term:`hotcue`
:hwlabel:`SHIFT` + :hwlabel:`1-4` numbered buttons  Clear hotcue
:hwlabel:`SHIFT` + Wheel nudge                      Fast search through track when not playing

> :hwlabel:`FLUX` button                              Enable slip mode (if shift is held down this decreases the range of the BPM slider, is play button is held down enter autoslip mode)

Reviewer: ```suggestion
:hwlabel:`FLUX` button                              Enable slip mode (if shift is held down this decreases the range of the BPM slider, if play button is held down enter autoslip mode)
```
What is autoslip mode? Isn't it problematic to need to hold down the play button for this because it would also start/stop the deck?

Reviewer 2: ```suggestion
:hwlabel:`FLUX` button                              Enable slip mode. If :hwlabel:`SHIFT` is held down this decreases the range of the BPM slider. If the :hwlabel:`PLAY` button is held down enter autoslip mode. 
```

Author: This is a bit confusing. If you look at the center of the mixer you see there's a button annoyingly labelled 'Play'
![image](https://user-images.githubusercontent.com/4432814/99892569-f21d5c80-2c6d-11eb-85bf-a238c759e654.png)

Reviewer 2: You can distinguish these by using something like:
- :hwlabel:`PLAY` (:hwlabel:`TRANSPORT` section)
- :hwlabel:`LOOP RECORDER PLAY`

The buttons at the top of each mixer column control which decks are routed to which effects units.

Mapping extras
~~~~~~~~~~~~~~~

> Autoslip mode: Holding down :hwlabel:`LOOP RECORDER PLAY` and pressing the :hwlabel:`FLUX` on a deck will enable autoslip mode on that deck. Turns on slip mode before doing certain actions and turns it off after the actions is finished to attempt to automatically engage slip mode. Works for beat loops, hot cues, and scratching (warning, can have weird effects when scratching).

Reviewer: ```suggestion
Autoslip mode: Holding down :hwlabel:`LOOP RECORDER PLAY` and pressing the :hwlabel:`FLUX` on a deck will enable autoslip mode on that deck. Autoslip turns on slip mode automatically before doing certain actions and turns it off after the actions is finished. It works for beat loops, hot cues, and scratching (warning, can have weird effects when scratching).
```
